include/zim/archive.h

```diff
@@ -51,6 +51,25 @@ namespace zim
    * An `Archive` is read-only, and internal states (as caches) are protected
    * from race-condition. Therefore, all methods of `Archive` are threadsafe.
    *
+   * Zim archives exist with two different namespace schemes: An old one and the new one.
+   * The method `hasNewNamespaceScheme` permit to know which namespace is used by the archive.
+   *
+   * When using old namespace, user entries may be stored in different namespaces (historically `A`, `I`, `J` or `-`).
+   * So path of the entries contains the namespace as a "subdirectory" : `A/foo.html`, `I/image.png`, ...
+   * All API taking or returing a path expect/will return a path with the namespace.
+   *
+   * When using new namespace, user entries are always stored whithout namespace.
+   * (For information, they are stored in the same namespace `C`. But consider there is no namespace as all API masks it)
+   * As there is no namespace, paths don't contain it: `foo.hmtl`, `image.png`, ...
+   * All API taking or returning a path expect/will return a path without namespace.
```

Comment:
This part will benefit from indentation:

```
When using old namespace scheme:
- blablabla
- blobloblo

When using new namespace scheme:
- blebleble
- bliblibli
```

Without such a visual hint, one can quickly read the sentence "All API taking or returning a path expect/will return a path with the namespace." while in search of some piece of information and  perceive it as a universally applicable statement.